### ISSUE TITLE
Improve formatting the title of the search results

### ISSUE THREAD

**Issue by blackgwe** [modernweb-dev/rocket#41]
The categories of the search results could be improved, e.g. using breadcrumb delimiters like "»".

Currently the following search terms are not rendered prettily:

**search for "small"**
```
t &gt; Why Rocket?
<strong>Small</strong>
```

**search for "frontend"**
```
Rocket &gt; Why Rocket?Small
```
<img width="490" alt="Bildschirmfoto 2021-01-22 um 07 30 25" src="https://user-images.githubusercontent.com/6592155/105455392-b4e04580-5c83-11eb-9463-e52c565a9100.png">

**Comment by daKmoR** [comment]
good catch - we planned to work on search improvements in a 2-3 weeks... 

if this is something of a priority for you then we can prioritise it for earlier - let us know 🤗

**Comment by Matsuuu** [comment]
Some improvements I would recommend, and could help implement 

- Make the results change cursor to `pointer` . (Twitter and Github do this)
- If the text content is cut off from the start, add a `...` before it.
- As @blackgwe suggested, use some kind of breadcrumb delimiter for all parts. One would be needed also between (in example image) the "Why Rocket?" and "Small" in my opinion.

**Comment by daKmoR** [comment]
indeed there are also a few other improvements we should make

https://github.com/daKmoR/rocket/issues/119

**Comment by Matsuuu** [comment]
I could maybe take a shot at applying these improvements :)

**Comment by Matsuuu** [comment]
I've added quite some of these fixes in a PR  [here](https://github.com/modernweb-dev/rocket/pull/77) . 

We might have to think about how to get the sub-titles formatted more nicely. A quick look at the code seemed to couple the "Why Rocket?" and "Small" onto the same object. 

Could we somehow split the subtitles into their own object?

**Comment by daKmoR** [comment]
Closing this old search UI issue as part of the repository reset.

Search is planned to return as a new `@rocket/js` docs/site feature, likely based on Pagefind. I’ll track the new implementation in #448. The breadcrumb/excerpt feedback from this issue is still useful and should be considered there.
